common/management/commands/project_external_updates.py

```diff
@@ -21,6 +57,87 @@ def handle(self, *args, **options):
                 print(traceback.format_exc())
                 pass
 
+def get_trello_board_id_from_url(url):
+    board_id = None
+    if url is not None:
+        # it will be in the format http[s]://trello.com/b/boardid/....
+        token = url.split('/')
+        
+        #board id will be at 4th position
+        #TODO : Check the URL using regex sample regex : http.://trello.com/b/(.*)[/]*.*
```

Comment:
Does this still need to happen?

Reply:
No need to use regex as board id always exists at this position.